main.html

```diff
@@ -1,77 +1,255 @@
+      These stories should describe the requested feature in terms agreeable to the guest team.
+      They also list any details from the host team on how the feature should be delivered in order for the work to be accepted.
+      Examples of such details include architecture constraints, coding conventions, dependency usages, data contracts, etc.
+    </li>
+    <li>
+      Trusted committers' time is made available to the guest team when the contributor is working.
+    </li>
+    <li>
+      Supported by the trusted committer(s), the contributor submits the pull request(s) to implement the requested feature.
+    </li>
+    <li>
+      Pull request is merged, and the new feature is completed.
+    </li>
+  </ol>
+</p>
+<p>
+  Note that these steps do not assume a specific system for the general organization of a team's time or priorities.
+  InnerSource assumes that teams already have existing such methods of organization.
+  InnerSource simply provides a framework of how to use them to work together where there is a guest team desiring to contribute code to a host.
+</p>
+
+<h2>
+  Benefits
+</h2>
+
+<p>
+  There are many benefits to collaborating via InnerSource.
+  First and foremost, <strong>the guest team gets their feature</strong> request when they need it without the long-term burden of maintaining it.
+  When the guest team would have spent time on the feature regardless (whether through <em>Workaround</em> or <em>Escalate</em>), InnerSource provides a way to use that time towards developing the feature in a place that's centrally-maintained and shared.
+</p>
+
+<p>
+  While that result is the shining benefit of InnerSource, another result that's easy to overlook is the fact that <strong>the host team gets their feature, too</strong>.
+  Recall that, as part of the InnerSource process, the product owner on the host team agrees from the outset that the requested feature is good and desirable.
+  InnerSource allows the host team to receive help in creating a better product for its consumers!
+</p>
+
+<p>
+  InnerSourcing provides the host team a scalable strategy for fulfilling the varying amounts of features requested from its many consumers.
+  Given the fixed capacity of the full-time members of the host team, it's likely that at times the combined business roadmaps of their consumers will require very (or even unreasonably) large amounts of work to be done in the host team's product(s).
+  Without InnerSource, this situation easily leads to a stressed, overworked team dealing with many feature requests escalated to its leaders.
+  However, if the host team operates via InnerSource, the engineering resources required to build those features will appear in-proportion to their importance in the form of guest contributors.
+  InnerSource becomes a force multiplier that allows a host team to temporarily act larger than its actual size during times of high demand.
```

Comment:
I like this point!  Is there a way to add emphasis in the reader to this with an example or something?